src/hera/v1/cron_workflow_service.py

```diff
@@ -0,0 +1,138 @@
+"""Holds the cron workflow service that supports client cron workflow submissions"""
+from typing import Tuple, Optional
+
+from argo.workflows.client import (
+    V1alpha1CronWorkflow,
+    CronWorkflowServiceApi,
+    V1alpha1CreateCronWorkflowRequest,
+    V1alpha1CronWorkflowSuspendRequest,
+    V1alpha1CronWorkflowResumeRequest,
+)
+from hera.v1.client import Client
+from hera.v1.config import Config
+
+
+class CronWorkflowService:
+    """Argo cron workflow service for performing actions against cron workflows - submissions, deletions, etc.
+
+    Parameters
+    ----------
+    domain: str
+        The Argo deployment domain to submit cron workflows to.
+    token: str
+        The token to use for authentication purposes. Note that this assumes the Argo deployment is fronted with a
+        deployment/service that can intercept a request and check the Bearer token.
+    namespace: str = 'default'
+        The K8S namespace the cron workflow service submits cron workflows to.
+        This defaults to the `default` namespace.
+    """
+
+    def __init__(self, domain: str, token: str, namespace: str = 'default'):
+        self._domain = domain
+        self._namespace = namespace
+        api_client = Client(Config(domain), token).api_client
+        self.service = CronWorkflowServiceApi(api_client=api_client)
+
+    def submit(self, cron_workflow: V1alpha1CronWorkflow, namespace: Optional[str] = None) -> V1alpha1CronWorkflow:
+        """Submits the given cron workflow to the given namespace.
+
+        Parameters
+        ----------
+        cron_workflow: V1alpha1CronWorkflow
+            The cron workflow to submit.
+        namespace: optional str
```

Comment:
Any chance the type specifications can use the `typing` package references? It provides neat consistency and clarity in documentation. It might also be nice to use something like auto-generated docs in the future to show these, and the types are nice 🙂 
```suggestion
        namespace: Optional[str] = None
```